ajax/libs/frappe-gantt/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "frappe-gantt",
+  "version": "0.0.7",
+  "filename": "frappe-gantt.min.js",
+  "description": "A simple, modern, interactive gantt library for the web",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/frappe/gantt.git"
+  },
+  "keywords": [
+    "gantt",
+    "svg",
+    "simple gantt",
+    "project timeline",
+    "interactive gantt",
+    "project management"
```

Comment:
Let's also add keywords shown in its git repo. (the block upper than commits, branches, releases, etc)